### ISSUE TITLE
Ignore whitespace changes

### ISSUE THREAD

**Issue by carloscz** [waderyan/vscode-gitblame#15]
Hi, it would be great if whitespace changes could be ignored. I have prepared change of git-blame enabling this behaviour, but original author of NPM package seems to be a bit inactive. 

Would You consider moving to another package for this feature?

**Comment by waderyan** [comment]
Interesting idea. Why do you want to ignore whitespace? I don't find it especially valuable to see blame on whitespace, but it isn't a distraction.

**Comment by carloscz** [comment]
yeah, but current version of git-blame seems to take whitespace changes into consideration. So if anyone replaces tabs for spaces in the entire file, vscode-gitblame marks him as author of every line :)

**Comment by waderyan** [comment]
Ah that is a good point. Want to do a PR? 😄

**Comment by carloscz** [comment]
I did for git-blame referenced by Your extension, but with no response from original developer :( I think I will wait few days more and then publish my extended module to NPM. Then it could be worked into this by PR

**Comment by carloscz** [comment]
Sorry for HUGE delay :( created pull request today https://github.com/waderyan/vscode-gitblame/pull/24 :)
